website/docs/whats-new/release-notes/partials/design-tokens.md

```diff
@@ -0,0 +1,82 @@
+ **`npm package: @hashicorp/design-system-tokens`**
+
+## 1.8.0
+
+**Month Day, Year**
+
+- `Design tokens` added color tokens for “Vault Secrets” product. 
+
+- `IconTile` updated component to include `vault-secrets` product option.
+
+[(#1640)](https://github.com/hashicorp/design-system/pull/1640) Thanks [@didoo](https://github.com/didoo)!
+
+## 1.7.0
+
+**Month Day, Year**
+
+- `marketing` added JSON output format for target (and in the process refactored internal logic for tokens generation). [(#1513)](https://github.com/hashicorp/design-system/pull/1513) Thanks [@dstaley](https://github.com/dstaley)! 
+
+## 1.6.0
+
+**Month Day, Year**
+
+- Add design token for loading state icon on search input. [(#1452)](https://github.com/hashicorp/design-system/pull/1452) Thanks [@alex-ju](https://github.com/alex-ju)! 
```

Comment:
We should have consistency in the tense used across the entries. I'll add a note to the instructions doc about this. 

```suggestion
- Added design token for loading state icon on search input. [#1452](https://github.com/hashicorp/design-system/pull/1452) Thanks [@alex-ju](https://github.com/alex-ju)! 
```